cmd/ncproxy/server_test.go

```diff
@@ -0,0 +1,1108 @@
+package main
```

Comment:
we have some for the tasks and internal requests under cmd/containerd-shim-runhcs-v1. I think it's fine since they're just unit tests. Though I wouldn't be opposed to moving some of this code out if you have suggestions on what would make sense.

Reply:
this is fine. I just wasn't sure.